apps/web/README.md

```diff
@@ -1,47 +1,15 @@
-# Astro Starter Kit: Minimal
+# Astro App
 
-```sh
-npm create astro@latest -- --template minimal
-```
+## Development
 
-[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/minimal)
-[![Open with CodeSandbox](https://assets.codesandbox.io/github/button-edit-lime.svg)](https://codesandbox.io/p/sandbox/github/withastro/astro/tree/latest/examples/minimal)
-[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/withastro/astro?devcontainer_path=.devcontainer/minimal/devcontainer.json)
+1. First follow the installation instructions in the repo [README](../../README.md).
+1. Then run `turbo dev` from the root directory to run all apps in dev mode. Note that running `turbo dev` from the app/web directory will only run dev mode for the Astro app, but some routes use `getStaticPaths`, which relies on data from Payload CMS. So the cms app also needs to be running locally in dev mode.
+1. Browse to http://localhost:4321/` to access the site.
 
-> 🧑‍🚀 **Seasoned astronaut?** Delete this file. Have fun!
+If using VS Code, be sure to install the recommended [Astro extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode), which provides language support for `.astro` files.
```

Comment:
Might be worth adding astro this repos's VSCode's [recommended plugins](https://code.visualstudio.com/docs/editor/extension-marketplace#_recommended-extensions) config too.

Reply:
+1

Reply:
Yep, it's in there already 👍